GLAA.Web/wwwroot/sass/glaa.scss

```diff
@@ -307,7 +307,7 @@ table.publicrreigster tbody td {
 }
 
 .public-register-selected-country {
-    background-image: url("/images/icon-button-remove.png");
+    background-image: url("/public/images/icon-button-remove.png");
```

Comment:
Ah! Good to see this fix 👍

Reply:
What I don't understand is why the build process in webpack didn't seem to pickup your image file properly